src/main/java/seedu/badMaths/trigograph/TrigoGraph.java

```diff
@@ -12,10 +12,18 @@ public void startGraphAnalysis() {
         TrigoGraphAnalyser analyser = new TrigoGraphAnalyser(trigoEqn);
         if(analyser.canStartAnalyser()){
             assert analyser.canStartAnalyser() == true;
-            Ui.printAmplitude(analyser.getAmplitude());
-            Ui.printFrequency(analyser.getFreq());
-            Ui.printPhase(analyser.getPhase());
-            Ui.printVerticalShift(analyser.getVerticalShift());
```

Comment:
The calling of Ui class can be refactored into a separate method. Since they are used quite often.